packages/cli/src/Server.ts

```diff
@@ -161,6 +161,8 @@ import { getLicense } from '@/License';
 import { licenseController } from './license/license.controller';
 import { corsMiddleware } from './middlewares/cors';
 
+import './events';
```

Comment:
can we remove this and the same line in `src/index.ts`, instead create an `init` function that sets up those 2 event listeners, and then call that function as late in the flow as possible. We have way too much code trying to set everything up as soon as possible, and that's the primary cause behind most of our memory woes right now.

Reply:
Sure thing!